project/Publishing.scala

```diff
@@ -97,7 +101,7 @@ object Publishing {
       cache = false,
       removeIntermediateContainers = BuildOptions.Remove.Always
     ),
-    ThisBuild / dockerCustomSettings := Nil, // setting the default value
+    ThisBuild / dockerCustomSettings := Nil // setting the default value
```

Comment:
TOL dangling commas are an error now? ugh...

Reply:
After 3+ years, I've given up on IntelliJ's scala plugin fixing its linting warnings for `project/*.scala` files. 😞

Trailing commas are valid in SBT, and preferred for source-controlled list items. But for developers, the false errors in the linter mask other issues.